src/ddl/covid_hosp.sql

```diff
@@ -334,12 +336,17 @@ version entitled "November 16, 2020 release 2.3".
   Sum of "total_staffed_adult_icu_beds" for hospitals reporting both
   "staffed_adult_icu_bed_occupancy" and "total_staffed_adult_icu_beds".
 
-NOTE: the following field is defined in the data dictionary but does not
-actually appear in the dataset.
+NOTES:
+There is a `date` column from the time series data which indicates the
+date corresponding to the values.
+For daily snapshot files, there is a `reporting_cutoff_start` value,
+defined as "Look back date start - The latest reports from each hospital
+is summed for this report starting with this date." We place this value
+into the `date` column.
 
-- `reporting_cutoff_start`
-  Look back date start - The latest reports from each hospital is summed for
-  this report starting with this date.
+We also add a column `record_type` that specifies if a row came from a
+time series file or a daily snapshot file. "T" = time series and
```

Comment:
i chose not to use a boolean in case we end up with more types, but if that's  too much overengineering we can juts do a bool. it's a char(1) so only a bit more space used

Reply:
char(1) is perfect